ee/vellum_ee/workflows/display/nodes/vellum/utils.py

```diff
@@ -93,4 +95,7 @@ def create_pointer(
             )
 
     vellum_variable_value = primitive_to_vellum_value(value)
-    return ConstantValuePointer(type="CONSTANT_VALUE", data=vellum_variable_value)
+    if pointer_type is InputVariablePointer:
+        return InputVariablePointer(type="INPUT_VARIABLE", data=InputVariableData(input_variable_id=value))
+    else:
```

Comment:
[q] Are we going to need to add the other types here?

Reply:
We definitely can to be safe but I have been slowly adding these types as these edge cases come up. Idk if there are going to be any use for them but I can add them preemptively anyway if you would like in a fast follow